canvas_sdk/v1/data/common.py

```diff
@@ -0,0 +1,72 @@
+class DocumentReviewMode:
```

Comment:
I think I asked a similar question on another PR, but I'll ask here too: Should we be using `TextChoices` here? At the very least it allows for defining these a little more succinctly, and also seems to be one of the standard ways to enumerate in Django.

Reply:
I'm fine with using `TextChoices`. I can change it. The linter doesn't like the dictionary format anyway.

Reply:
I can see a dictionary format being more readable, though TextChoices does come with some additional properties on it that might be convenient for users: choices, labels, values, and names.